src/8800.c

```diff
@@ -78,7 +79,147 @@ void update_cameras(void) {
+
+                ulx = camera->viewportStartX;
+                uly = camera->viewportStartY;
+                lrx = ulx + camera->viewportW;
+                lry = uly + camera->viewportH;
+
+                if (ulx < 0) {
+                    ulx = 0;
+                }
+                if (uly < 0) {
+                    uly = 0;
+                }
+                if (lrx < 1) {
+                    lrx = 1;
+                }
+                if (lry < 1) {
+                    lry = 1;
+                }
+
+                if (ulx > SCREEN_WIDTH - 1) {
+                    ulx = SCREEN_WIDTH - 1;
+                }
+                if (uly > SCREEN_HEIGHT - 1) {
+                    uly = SCREEN_HEIGHT - 1;
+                }
+                if (lrx > SCREEN_WIDTH) {
+                    lrx = SCREEN_WIDTH;
+                }
+                if (lry > SCREEN_HEIGHT) {
+                    lry = SCREEN_HEIGHT;
+                }
+
+                gDPSetScissor(gMasterGfxPos++, G_SC_NON_INTERLACE, ulx, uly, lrx, lry);
+                gDPSetTextureLOD(gMasterGfxPos++, G_TL_TILE);
+                gDPSetTextureLUT(gMasterGfxPos++, G_TT_NONE);
+                gDPSetTextureDetail(gMasterGfxPos++, G_TD_CLAMP);
+                gDPSetTexturePersp(gMasterGfxPos++, G_TP_PERSP);
+                gDPSetTextureFilter(gMasterGfxPos++, G_TF_BILERP);
+                gDPSetTextureConvert(gMasterGfxPos++, G_TC_FILT);
+                gDPSetCombineMode(gMasterGfxPos++, G_CC_SHADE, G_CC_SHADE);
+                gDPSetCombineKey(gMasterGfxPos++, G_CK_NONE);
+                gDPSetAlphaCompare(gMasterGfxPos++, G_AC_NONE);
+                gDPSetRenderMode(gMasterGfxPos++, G_RM_OPA_SURF, G_RM_OPA_SURF2);
+                gDPSetColorDither(gMasterGfxPos++, G_CD_DISABLE);
+                gSPClipRatio(gMasterGfxPos++, FRUSTRATIO_2);
+                gDPSetColorImage(gMasterGfxPos++, G_IM_FMT_RGBA, G_IM_SIZ_16b, SCREEN_WIDTH,
+                                 osVirtualToPhysical(nuGfxCfb_ptr));
+                gDPPipeSync(gMasterGfxPos++);
+
+                if (!(camera->flags & 0x10)) {
+                    gSPPerspNormalize(gMasterGfxPos++, camera->perspNorm);
+                }
+
+                guMtxF2L(camera->perspectiveMatrix, &gDisplayContext->camPerspMatrix[gCurrentCamID]);
+                gSPMatrix(gMasterGfxPos++, &gDisplayContext->camPerspMatrix[gCurrentCamID], G_MTX_NOPUSH | G_MTX_LOAD |
+                          G_MTX_PROJECTION);
+            }
+
+            camera->unkMatrix = &gDisplayContext->matrixStack[gMatrixListPos];
+            matrixListPos = gMatrixListPos++;
+            guRotate(&gDisplayContext->matrixStack[matrixListPos], -camera->trueRotation.x, 0.0f, 1.0f, 0.0f);
+            camera->vpAlt.vp.vtrans[0] = camera->viewport.vp.vtrans[0] + gGameStatusPtr->unk_82;
+            camera->vpAlt.vp.vtrans[1] = camera->viewport.vp.vtrans[1] + gGameStatusPtr->unk_83;
+
+            if (!(camera->flags & 0x10)) {
```

Comment:
Looking at the else block for when this flag is set it might be more accurate to call it CAM_FLAG_HUD or something since it skips rendering of the player, entities, etc.